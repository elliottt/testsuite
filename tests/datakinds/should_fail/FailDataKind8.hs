{-# LANGUAGE DataKinds #-}
module FailDataKind8 where

data type Nat = Ze | Su Nat

data kind Test = Foo Nat
