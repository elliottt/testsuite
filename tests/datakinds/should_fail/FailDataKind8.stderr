
FailDataKind8.hs:6:22:
    ‛Nat’ of kind ‛*’ is not promotable
    In the kind ‛Nat’
